test/integration/discovery-consul.bats

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bats
+
+load helpers
+
+# Address on which Consul will listen (random port between 7000 and 8000).
```

Comment:
You should pick a different range since you might conflict with `zookeeper` which is also using that port range.